test/onnx/expect/TestOperators.test_conv.expect

```diff
@@ -1,6 +1,6 @@
 ir_version: 6
 producer_name: "pytorch"
-producer_version: "XXX"
+producer_version: "1.7"
```

Comment:
Related to commit 2e6ee853abe8b. @mattip are we attached to keeping `XXX`? If so, I think your fix to this was incomplete, because things like `python test/onnx/test_operators.py --accept` will re-introduce version numbers.

Reply:
Fixed in gh-41910

Reply:
@hameerabbasi it will be better if you undo these hunks. If you don't, then this PR will conflict with mattip's.